packages/lib/src/components/components-map.ts

```diff
@@ -182,6 +182,7 @@ export const ComponentsMap = {
     [TxVariants.klarna]: Klarna,
     [TxVariants.klarna_account]: Klarna,
     [TxVariants.klarna_paynow]: Klarna,
+    [TxVariants.klarna_b2b]: Klarna,
```

Comment:
Was `klarna_b2b` a redirect component before?

Reply:
No I think it should link to `Klarna` component ([v5](https://github.com/Adyen/adyen-web/blob/aa18474e2788adb8a86abbe853a4904ee2e31d28/packages/lib/src/components/index.ts#L201)).